### DISCUSSION TITLE
Payment Link

### DISCUSSION THREAD

**Discussion by euluizaraujo** [juspay/hyperswitch#3069] [Q&A] [answered]
Hello, when is the payment link expected to be 100% available opensource?  It is a resource that would help us a lot.

**Comment by SanchithHegde** [comment, marked as answer]
Hey @euluizaraujo, the payment links feature is under development, we're expecting it would be available in January 2024.

**Comment by AmanMauryaXeno** [comment]
Hey @euluizaraujo, if you're open to share, wanted to understand the following things from you w.r.t payment links;

1. For what's your use case for payment links?
2. What are you currently using to send payment links? (If any) Are there any gaps in it?
3. What are you looking forward too in our open sourced version?
